core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -83,11 +83,9 @@ export class CreateServiceComponent implements OnInit {
   errMessage: any;
   invalidServiceName:boolean=false;
   invalidDomainName:boolean=false;
-  public apiDeployment = "aws_apigateway";
-  public functionDeployment = "aws_lambda";
-  public websiteDeployment = "aws_cloudfront";
-  public buildEnvironment = environment;
-  
+  public buildEnvironment:any = environment;
+  public deploymentTargets = this.buildEnvironment["INSTALLER_VARS"]["CREATE_SERVICE"]["DEPLOYMENT_TARGETS"];
+  public selectedDeploymentTarget = "";
```

Comment:
this is empty. where are you setting it

Reply:
It is two way bound to the result of the deployment-target radio-checkbox, so in the html